<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>65535</width>
    <height>65535</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Sharingan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic/icon/icon.qrc">
    <normaloff>:/new/prefix1/Network.ico</normaloff>:/new/prefix1/Network.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="labelShow">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>462</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/background-image/player.png);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>333</y>
     <width>466</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>50</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButtonOpenPic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(5, 200, 21);</string>
      </property>
      <property name="text">
       <string>打开图片</string>
      </property>
      <property name="icon">
       <iconset resource="../pic/icon/icon.qrc">
        <normaloff>:/new/prefix1/Folder.ico</normaloff>:/new/prefix1/Folder.ico</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOpenMov">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(48, 160, 200);</string>
      </property>
      <property name="text">
       <string>打开动态图</string>
      </property>
      <property name="icon">
       <iconset resource="../pic/icon/icon.qrc">
        <normaloff>:/new/prefix1/Movies 2.ico</normaloff>:/new/prefix1/Movies 2.ico</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 200, 7);</string>
      </property>
      <property name="text">
       <string>播放</string>
      </property>
      <property name="icon">
       <iconset resource="../pic/icon/icon.qrc">
        <normaloff>:/new/prefix1/Sharingan.ico</normaloff>:/new/prefix1/Sharingan.ico</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 58, 40);</string>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
      <property name="icon">
       <iconset resource="../pic/icon/icon.qrc">
        <normaloff>:/new/prefix1/Pictures (2).ico</normaloff>:/new/prefix1/Pictures (2).ico</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../pic/icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
